modules/dataframe/include/inviwo/dataframe/datastructures/dataframe.h

```diff
@@ -136,6 +136,8 @@ class IVW_MODULE_DATAFRAME_API DataFrame {
      */
     std::shared_ptr<Column> getColumn(size_t index);
     std::shared_ptr<const Column> getColumn(size_t index) const;
+    std::shared_ptr<Column> getColumn(const std::string &name);
```

Comment:
State what happens when
1. No column with that name exist
2. More than one column has the same name